pkg/metadata/inventories/inventories.go

```diff
@@ -110,6 +110,7 @@ const (
 
 	// System Probe general config values
 	AgentSPOOMKillEnabled                AgentMetadataName = "feature_oom_kill_enabled"
+	AgentSPWinCrashEnabled               AgentMetadataName = "feature_windows_crash_detect_enabled"
```

Comment:
nit: Use full name `windows_crash_detection` or the module name `wincrashdetect` here